ana_scans.py

```diff
@@ -0,0 +1,227 @@
+#!/bin/env python
+
+def launchAna(args):
+  return launchAnaArgs(*args)
+
+def launchAnaArgs(anaType, cName, cType, scandate1, scandate2=None, ztrim=4.0, chConfigKnown=False, channels=False, panasonic=False):
+  import os
+  import subprocess
+  from subprocess import CalledProcessError
+  from anaInfo import ana_config
+  from gempython.utils.wrappers import runCommand
+
+  dataPath  = os.getenv('DATA_PATH')
+  dirPath   = ""
+  elogPath  = "%s/%s"%(os.getenv('ELOG_PATH'),scandate1)
+    
+  print "Analysis Requested: %s"%(anaType)
+
+  #Build Commands
+  cmd = ["python"]
```

Comment:
You are setting it up in such a way that the execution of the script from the subshell is invoked via:
```bash
python <super long full path to script locatin> <options>
```
What I am requesting changes your operation procedure to do:
```bash
<script> <options>
```

>  If so shouldn't there just be a line added to:
`cmsgemos/setup/paths.sh`
> 

No, this repository is `vfatqc-python-scripts` and it is this location that needs to be added to the `$PATH`, *not* anything in `cmsgemos`